planning/autoware_obstacle_stop_planner/schema/obstacle_stop_planner.schema.json

```
@@ -0,0 +1,322 @@
+                  "default": "1.0"
+                },
+                "pedestrian_lateral_margin": {
+                  "type": "number",
+                  "description": "offset from pedestrian side edge for expanding the search area of the surrounding point cloud [m]",
+                  "default": "1.0"
+                },
+                "unknown_lateral_margin": {
+                  "type": "number",
+                  "description": "offset from unknown side edge for expanding the search area of the surrounding point cloud [m]",
+                  "default": "1.0"
+                }
+              },
+              "required": [
+                "lateral_margin",
+                "vehicle_lateral_margin",
+                "pedestrian_lateral_margin",
+                "unknown_lateral_margin"
+              ]
+            },
+            "target_velocity": {
+              "type": "object",
+              "properties": {
+                "max_slow_down_velocity": {
+                  "type": "number",
+                  "description": "max slow down velocity (use this param if consider_constraints is False)[m/s]",
+                  "default": "1.38 "
+                },
+                "min_slow_down_velocity": {
+                  "type": "number",
+                  "description": "offset from vehicle side edge for expanding the search area of the surrounding point cloud [m]",
+                  "default": "0.28"
+                },
+                "slow_down_velocity": {
+                  "type": "number",
+                  "description": "target slow down velocity (use this param if consider_constraints is True)[m/s]",
+                  "default": "1.38"
+                }
+              },
+              "required": ["max_slow_down_velocity", "min_slow_down_velocity", "slow_down_velocity"]
+            },
+            "constraints": {
+              "type": "object",
+              "properties": {
+                "jerk_min_slow_down": {
+                  "type": "number",
+                  "description": "min slow down jerk constraint [m/sss]",
+                  "default": "-0.3"
+                },
+                "jerk_span": {
+                  "type": "number",
+                  "description": "fineness param for planning deceleration jerk [m/sss]",
+                  "default": "-0.01"
+                },
+                "jerk_start": {
+                  "type": "number",
+                  "description": "init jerk used for deceleration planning [m/sss]",
+                  "default": "-0.1"
+                }
+              },
+              "required": ["jerk_min_slow_down", "jerk_span", "jerk_start"]
+            },
+            "consider_constraints": {
+              "type": "boolean",
+              "description": "set 'True', if no decel plan found under jerk/dec constrains, relax target slow down vel [-]",
+              "default": "False"
```

Could you make it to lower case?
It seems it causes some error when converting to markdown if false/true are capitalized.
```suggestion
              "default": "false"
```